example/App.js

```diff
@@ -53,93 +68,207 @@ const App = () => {
     setDisplay('spinner');
   };
 
-  return (
-    <>
-      <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
-        <ScrollView
-          contentInsetAdjustmentBehavior="automatic"
-          style={styles.scrollView}>
-          <Header />
-          {global.HermesInternal !== null && (
-            <View style={styles.engine}>
-              <Text style={styles.footer}>Engine: Hermes</Text>
-            </View>
-          )}
-          <View style={styles.body}>
-            <View testID="appRootView" style={styles.container}>
-              <View style={styles.header}>
-                <Text style={styles.text}>Example DateTime Picker</Text>
-              </View>
-              <View style={styles.header}>
-                <Text style={{margin: 10, flex: 1}}>text color (iOS only)</Text>
-                <TextInput
-                  value={color}
-                  style={{height: 60, flex: 1}}
-                  onChangeText={text => {
-                    setColor(text.toLowerCase());
-                  }}
-                  placeholder="color"
-                />
-              </View>
-              <View style={styles.button}>
-                <Button
-                  testID="datePickerButton"
-                  onPress={showDatepicker}
-                  title="Show date picker default!"
-                />
-              </View>
-              <View style={styles.button}>
-                <Button
-                  testID="datePickerButtonSpinner"
-                  onPress={showDatepickerSpinner}
-                  title="Show date picker spinner!"
-                />
+  if (Platform.OS !== 'windows') {
+    return (
```

Comment:
Can we share more between platforms?

Reply:
The example app demoes a few modes that we don't support - time picker (not yet) and spinner. Thought it'd be easier to just add 1 branch vs exclude all those tags for windows.